screenpy/resolutions/custom_matchers/is_in_bounds.py

```diff
@@ -62,16 +74,20 @@ def describe_mismatch(self, item: float, mismatch_description: Description) -> N
         )
 
 
-def is_in_bounds(*bounds: int | (float | str)) -> IsInBounds:
+def is_in_bounds(*bounds: int | float | str) -> IsInBounds:
```

Comment:
I really wish i could tell you what i was thinking here. Hahaha.

Reply:
It's technically correct either way.  I'll be honest, I don't remember fixing this one.  I thought ruff might have automatically done it, but that doesn't seem to be the case.